src/main/java/org/heigit/ohsome/ohsomeapi/inputprocessing/InputProcessor.java

```diff
@@ -867,4 +870,8 @@ public boolean includeOSMMetadata() {
   public boolean isUnclipped() {
     return unclipped;
   }
+  
+  public boolean modifications() {
```

Comment:
suggestion to keep a more consistent naming schema the getter method of the modifications boolean should be named isModifications()!

Reply:
Yeah I was thinking about that one how to name it properly.. it's true that a boolean getter should somehow have "is" or "has" in it's name. I decided to not go for "hasModifications()" as the response always has the modifications in it, no matter if this property is given or not. "isModifications()" does also not fit that well in my opinion..
What about something like "hasModificationsOnly()" and then also naming the boolean "modificationsOnly" but keep the name of the property inside the request "modifications" for simplicity purpose?

Reply:
Yes this sounds better, hasModificationsOnly() or maybe  better than this, as  this is a option of the query and you could ask for it like, "is this query for modifications only" , isModificationsOnly().